src/thread.ml

```diff
@@ -29,7 +29,7 @@ let globals t = t.globals
 let solver_mod : Solver.t solver_module = (module Solver)
 
 let create () =
-  let solver = S (solver_mod, Solver.create ~logic:QF_BVFP ()) in
+  let solver = S (solver_mod, Solver.create ~logic:QF_BV ()) in
```

Comment:
Is this intentional? I'm only worried that when mixing floats with bvs Z3 will return `Unknown` as there's no logic to solve FPA

Reply:
Oops sorry. For some reason I was getting an error on latest encoding and changed this without knowing what I was doing. :sweat_smile: